src/renderer/components/layout/main-layout-header.tsx

```diff
@@ -1,39 +1,17 @@
 import React from "react";
 import { observer } from "mobx-react";
-
-import { ClusterActions } from "../cluster-manager";
 import { Cluster } from "../../../main/cluster";
 import { cssNames } from "../../utils";
-import { MenuActions, MenuItem } from "../menu";
 
 interface Props {
   cluster: Cluster
   className?: string
 }
 
 export const MainLayoutHeader = observer(({ cluster, className }: Props) => {
-  const actions = ClusterActions(cluster);
-  const renderMenu = () => (
-    <MenuActions autoCloseOnSelect className="ClusterActionsMenu">
```

Comment:
Why do we no longer want this, to specific to Kube Clusters?

Reply:
I'm not sure if we need it anymore now that we already have menu in catalog & hotbar items. If we want to bring this back then it should be dynamic like those other menus.